### PR TITLE
feat(pattern/hero): add hero design

### PR DESCRIPTION
**Pull request by nherwegh**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [X] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [X] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

File: src/docs/Components/Hero/index.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: 'Hero'
+order: 1
+description: 'A hero is the first thing your user sees when the page loads. This is a zone that you can use to draw the  attention of the user on a specific message.'
```
Comment (C0ZEN) [inline src/docs/Components/Hero/index.mdx:4]: Two spaces between `the  attention`.

File: src/docs/Components/Hero/index.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+status:
+  sketch: 'bêta'
+  scss: 'bêta'
+---
+
+> A hero is the first thing your user sees when the page loads. This is a zone that you can use to draw the attention of the user on a specific message.
```
Comment (caqueste) [inline src/docs/Components/Hero/index.mdx:10]: I suggest:
A hero is an element that you can use on the top of your pages add visual ambience or draw the user's attention on a specific message.

File: src/docs/Components/Hero/index.mdx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Variations
+
+We want to provide as many cases as possible so we created three different versions of heroes.
+
+### Title only hero
+
+This is the most minimal hero case you will find in Mozaïc Design System. This version is best used when you need to introduce a page with just an image and a title on top of it.
```
Comment (caqueste) [inline src/docs/Components/Hero/index.mdx:18]: Remove `you will find in Mozaïc Design System`. I think this part is useless.

File: src/docs/Components/Hero/index.mdx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+--title only preview--
+
+### Title with text hero
```
Comment (caqueste) [inline src/docs/Components/Hero/index.mdx:22]: Maybe we can merge this part this the previous one, and indicate that the text is optional.

File: src/docs/Components/Hero/index.mdx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+This is the most common hero case we recommend to use. This version is made of a headline, a text content and a background image.
+
+--title with text preview--
+
+### Title with text and boxed content
+
+Sometimes, you need to take some content apart of the overall page layout. That is why we provide a hero version with a boxed content either centered under the text content or aligned to the right.
```
Comment (caqueste) [inline src/docs/Components/Hero/index.mdx:30]: - Replace `Sometimes, ...` by `In specific cases, ...`
- ~~...you need to take some content apart of the overall page layout.~~ → ...you need to take some content apart of the overall page layout, for example, to put forward a feature.

File: src/docs/Components/Hero/index.mdx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+<Highlight theme="warning" title="Be aware">
+
+Please note that a hero must not be based on the image but on the text content.
+
+</Highlight>
+
+## Do's and don'ts
+
+<HintItem dont>
+  Don't use less than G800 for the overlay grey value.
```
Comment (caqueste) [inline src/docs/Components/Hero/index.mdx:43]: To have better readable content, don't use less than G800...

File: src/docs/Components/Hero/index.mdx (hunk 2 above)
Comment (nherwegh) [reply src/docs/Components/Hero/index.mdx:10]: good, thx

File: src/docs/Components/Hero/index.mdx (hunk 4 above)
Comment (nherwegh) [reply src/docs/Components/Hero/index.mdx:22]: I think that I can be useful to distinguish these two cases

**Review by caqueste — APPROVED** [review]

**Review by tiloyi — APPROVED** [review]

**Pull request merged**